src/js/edu-benefits/components/education-history/EducationPeriod.jsx

```diff
@@ -64,7 +64,7 @@ export default class EducationPeriod extends React.Component {
             options={hoursTypes}
             onValueChange={(update) => {onValueChange('hoursType', update);}}/>
         <ErrorableTextInput
-            label="Degree, diploma or certificate received"
+            label="Name of degree, diploma, or certificate"
```

Comment:
Whenever the labels change, we also need to change them on the review page for these questions. There should be a corresponding review fields file for each section, so check there for the labels that need to change there as well.